### PR TITLE
PIM-9923: Merge 5.0 into Master

### PR DESCRIPTION
**Pull request by anaisbaunelemaire**
/!\ The lib upgrade will be done in another PR.

**Definition Of Done (for Core Developer only)**

- [ ] Tests
- [ ] Migration & Installer
- [ ] PM Validation (Story)
- [ ] Changelog (maintenance bug fixes)
- [ ] Tech Doc

### CODE REVIEW

**Review by mmetayer — APPROVED** [review]

**Review by pierrejolly — APPROVED** [review]

File: src/Akeneo/Tool/Component/Connector/Writer/GlobalFunctionsHelper.php [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+declare(strict_types=1);
+
+namespace Akeneo\Tool\Component\Connector\Writer;
+
+use Box\Spout\Common\Helper\GlobalFunctionsHelper as SpoutGlobalFunctionsHelper;
+
+/**
+ * @todo CPM-248 remove this class
+ */
+class GlobalFunctionsHelper extends SpoutGlobalFunctionsHelper
```
Comment (nmarniesse) [inline src/Akeneo/Tool/Component/Connector/Writer/GlobalFunctionsHelper.php:12]: class to remove

File: src/Akeneo/Tool/Component/Connector/Writer/File/FlatItemBufferFlusher.php [hunk 2]
```diff
@@ -4,8 +4,8 @@
 
 use Akeneo\Tool\Component\Batch\Model\StepExecution;
 use Akeneo\Tool\Component\Batch\Step\StepExecutionAwareInterface;
+use Akeneo\Tool\Component\Connector\Writer\WriterFactory;
```
Comment (nmarniesse) [inline src/Akeneo/Tool/Component/Connector/Writer/File/FlatItemBufferFlusher.php:7]: to revert

**Review by nmarniesse — APPROVED** [review]
I didn't see the clean will be done in a separate ticket. GTM

**Comment by anaisbaunelemaire** [comment]
warm

**Pull request merged**